ui/tests/e2e/test.js

```diff
@@ -239,8 +235,9 @@ describe("End-to-end", () => {
   /**
    * Waits for facet card to be rendered.
    */
-  async function waitForFacetCard(facetName) {
+  async function waitForFacetCard(facetName, facetValue) {
```

Comment:
Why did this change? Just passing "Avuncular" should work.. If it doesn't work, something else needs to be fixed